### ISSUE TITLE
Implement functionality for HSL and HSLA formats 

### ISSUE THREAD

**Issue by jdwilkin4** [DBenMoshe/contrast-ratio-repo#37]
As a user I should be able to input a HSL format and have the correct contrast ratio result displayed. 
We want to mimic the functionality here:
<img width="1046" alt="Screen Shot 2022-08-08 at 6 57 26 PM" src="https://user-images.githubusercontent.com/67210629/183546777-97bb5280-358d-4627-a718-e4e1b8cca11d.png">

Don't worry about styling.
Just focus on functionality right now.

**Comment by AndreyDmitriev39r** [comment]
I don't even remember how to use this color format properly.
Good opportunity to refresh the basic knowledge, so I am on this one.
